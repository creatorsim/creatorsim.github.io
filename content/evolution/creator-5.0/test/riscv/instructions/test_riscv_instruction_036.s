
#
# Creator (https://creatorsim.github.io/creator/)
#

.text
main:

    li x5, 10
    li x6, 13
    li x7, 45
    li x8, -2
    
    mulhsu x9, x5, x6
    mulhsu x10, x6, x7
    mulhsu x11, x7, x8
    mulhsu x12, x8, x5