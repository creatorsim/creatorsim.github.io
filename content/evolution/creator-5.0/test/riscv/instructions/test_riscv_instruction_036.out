
cr[PC]:0x20; ir[x5,t0]:0xa; ir[x6,t1]:0xd; ir[x7,t2]:0x2d; ir[x8,fp,s0]:0xfffffffe; ir[x9,s1]:0x82; ir[x10,a0]:0x249; ir[x11,a1]:0xffffffa6; ir[x12,a2]:0xffffffec; keyboard[0x0]:''; display[0x0]:''; 

